src/lib/arch/aarch64/Instruction_execute.cc

```diff
@@ -1663,8 +1667,10 @@ void Instruction::execute() {
         return executionNYI();
         break;
       }
-      case Opcode::AArch64_CMHSv16i8: {
-        return executionNYI();
+      case Opcode::AArch64_CMHSv16i8: {  // cmhs vd.16b, vn.16b, vm.16b
+        results[0] = neonHelp::vecCompare<uint8_t, 16>(
+            operands, false,
+            [](uint8_t x, uint8_t y) -> bool { return (x >= y); });
```

Comment:
I think Comparisons should be signed unless specified otherwise in the ISA profile